docs/source/guide/explanation/algorithms/segmentation/instance_segmentation.rst

```diff
@@ -39,8 +39,8 @@ For the supervised training we use the following algorithms components:
 Dataset Format
 **************
 
-For the dataset handling inside OTX, we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_. For instance segmentation we support `COCO <https://cocodataset.org/#format-data>`_ and `Pascal-VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_ dataset formats.
-If you have your dataset in those formats, then you can simply run OTX using one line of code:
+For the dataset handling inside OpenVINO™ Training Extensions, we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_. For instance segmentation we support `COCO <https://cocodataset.org/#format-data>`_ and `Pascal-VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_ dataset formats.
+If you have your dataset in those formats, then you can simply run OpenVINO™ Training Extensions using one line of code:
```

Comment:
OpenVINO™ Training Extensions -> OTX  is better?

Reply:
it is better to keep as 'then you can simply run using one line of code' not mentioning OTX. Isn't it?

Reply:
I ment,
before: then you can simply run OpenVINO™ Training Extensions using one line of code:
after: then you can simply run OTX using one line of code:

Reply:
It's explanation in doc. Is it okay?